spine-ts/core/src/AssetManager.ts

```diff
@@ -131,8 +131,12 @@ module spine {
 
 			img.crossOrigin = "anonymous";
 			img.onload = (ev) => {
-				if (bitmapOptions) {
+				// if not in Firefox and not in IE and bitmapOptions call createImageBitmap
+				if (bitmapOptions &&
+				    !(navigator.userAgent.toLowerCase().indexOf('firefox') > -1) &&
+				    !(navigator.userAgent.toLowerCase().indexOf('Trident') > -1) ) {
 					var this_  = this;
+

```

Comment:
I would add this to a function or constant, just in case we need to add/remove some browsers

Reply:
I was looking also for a place to put a function `isFirefox() `and `isIE()` but there is not such a "utils" file, also checked in the source base and in other places they do it like that 😢  . 
So in order to make the changes less intrusive to the spine guys I decided to leave it like that...